src/core/parcel-validation.ts

```diff
@@ -0,0 +1,127 @@
+import { generateLazyValidator, JSONSchema, ValidateFunction } from '../validation'
+import { isOutOfBoundsParcel } from './parcel-exceptions'
+
+/**
+ * Represents a parcel with X and Y coordinates
+ * @public
+ */
+export type Parcel = { x: number; y: number }
+
+/**
+ * Standard limits for parcels in Decentraland
+ * @public
+ */
+export const PARCEL_LIMITS = {
+  minX: -150,
+  maxX: 150,
+  minY: -150,
+  maxY: 150
+}
+
+/**
+ * Namespace with utilities to validate parcels
+ * @public
+ */
+export namespace Parcel {
+  export const schema: JSONSchema<Parcel> = {
+    type: 'object',
+    required: ['x', 'y'],
+    properties: {
+      x: {
+        type: 'number'
+      },
+      y: {
+        type: 'number'
+      }
+    }
+  }
+
+  const schemaValidator: ValidateFunction<Parcel> = generateLazyValidator(schema)
+
+  // Converts coordinates to string in "x,y" format
+  export function parcelToString({ x, y }: Parcel): string {
+    return `${x},${y}`
+  }
+
+  // Converts a string in "x,y" format to a Parcel object
+  export function stringToParcel(position: string): Parcel | null {
+    const match = position.match(/^(-?\d+),(-?\d+)$/)
+    if (!match) return null
+
+    const [, xStr, yStr] = match
+    const x = parseInt(xStr, 10)
+    const y = parseInt(yStr, 10)
+
+    return { x, y }
+  }
+
+  /**
+   * Validates if x,y coordinates form a valid parcel according to the schema
+   */
+  export const validate: ValidateFunction<Parcel> = (parcel: any): parcel is Parcel => {
+    return schemaValidator(parcel)
+  }
+
+  /**
+   * Validates if a string in "x,y" format represents a valid parcel
+   */
+  export function validateParcelString(parcelString: string): boolean {
```

Comment:
Following the naming convention of other methods here:
```suggestion
  export function isParcelStringValid(parcelString: string): boolean {
```